res/Resources.php

```diff
@@ -186,6 +186,15 @@
 		'dependencies' => 'jquery.ui.autocomplete',
 		'targets' => array( 'mobile', 'desktop' )
 	),
+	
+	// Purge resources
+	'ext.smw.purge' => $moduleTemplate + array(
+		'scripts' => 'smw/util/ext.smw.util.purge.js',
+		'messages' => array(
+			'smw-purge-failed'
+		)
+	),
```

Comment:
> I had the feeling that "targets"

... can be added, more importantly is `'position' => 'top',` so that the module is loaded as soon as possible.